integration-tests/rest-tests-jaxrs/pom.xml

```diff
@@ -203,6 +205,22 @@
                       <commmand>
                         /subsystem=logging/periodic-rotating-file-handler=FILE:write-attribute(name="file",value={path="${project.build.directory}/wildfly-test.log"})
                       </commmand>
+                      <!-- caches -->
+                      <commmand>
+                        /subsystem=infinispan/cache-container=hawkular-alerts/:add(module=org.jboss.as.clustering.infinispan,start=LAZY)
```

Comment:
It works fine doing it this way.  It was not east to use the xsl transformer approach given the existing test setup in the pom.